.github/workflows/autobuild.yml

```diff
@@ -114,39 +127,25 @@ jobs:
-
-      # There might be two assets (at least for Debian)
-      - name:                       Upload Release Asset 2
-        if:                         matrix.config.asset_name_two != null
-        id:                         upload-release-asset-two
-        uses:                       actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN:             ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url:               ${{ needs.create_release.outputs.upload_url }}
-          asset_path:               ${{ matrix.config.asset_path_two }}
-          asset_name:               ${{ matrix.config.asset_name_two }}
-          asset_content_type:       application/octet-stream
-
-      ## Upload assets latest release
-      - name:                       Upload latest Release Asset 1
-        id:                         upload-latest-release-asset
-        uses:                       actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN:             ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url:               ${{ needs.create_release.outputs.upload_latest_url }} # See this blog post for more info: https://jasonet.co/posts/new-features-of-github-actions/#passing-data-to-future-steps
-          asset_path:               ${{ matrix.config.asset_path }}
-          asset_name:               ${{ matrix.config.asset_latest_name }}
-          asset_content_type:       application/octet-stream
-      - name:                       Upload latest Release Asset 2
-        if:                         matrix.config.asset_latest_name_two != null
-        id:                         upload-latest-release-asset-two
-        uses:                       actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN:             ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url:               ${{ needs.create_release.outputs.upload_latest_url }}
-          asset_path:               ${{ matrix.config.asset_path_two }}
-          asset_name:               ${{ matrix.config.asset_latest_name_two }}
-          asset_content_type:       application/octet-stream
+          
+  ## Builds a Linux flatpack ##
+  flatpak-builder:
+    name: "Flatpak Builder"
+    runs-on: ubuntu-latest
+    needs: [create_release]
+    container:
+      image: docker.io/bilelmoussaoui/flatpak-github-actions
+      options: --privileged
+    steps:
+    - uses: actions/checkout@v2
+    - name:                    Change branch name in manifest
+      run:                     python3 io.jamulus.Jamulus.prepare.py $GITHUB_SHA
+      working-directory:       ./distributions/
+    - uses: bilelmoussaoui/flatpak-github-actions@v2
+      with:
+        bundle: "io.jamulus.Jamulus.flatpak"
+        manifest-path: "distributions/io.jamulus.Jamulus.json"
+    - uses: actions/download-artifact@v2
+      with:
+        name: io.jamulus.Jamulus
+        path: ~/io.jamulus.Jamulus
```

Comment:
The upload step is still missing. Can you look at my AndroidDev branch?

Reply:
will do.
this only is about Flatpak, right?

Reply:
Yes.